Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "fslock"
-version = "0.2.1"
-authors = ["brunoczim <brunoczim@gmail.com>"]
-edition = "2018"
+version = "0.2.2"
+authors = ["brunoczim <brunoczim@gmail.com>", "Fredrick Brennan <copypaste@kittens.ph>"]
```

Comment:
This seems a little weird... I mean, isn't it too soon for this? And I actually was planning remove author field

Reply:
Oh, I'm sorry, I can remove this, I didn't think you'd reply so this is a commit for the fork and wasn't supposed to be in the PR